README.md

```diff
@@ -1,40 +1,40 @@
-* stored value
-* data protection
-
-## Requirements
-
+This is the officially supported Java library for using Adyen's APIs.
+ 
+The library supports all APIs under the following services:
+ 
+* [Checkout API](https://docs.adyen.com/api-explorer/#/CheckoutService/v67/overview): Our latest integration for accepting online payments. Current supported version: **v67**
+* [Payments API](https://docs.adyen.com/api-explorer/#/Payment/v52/overview): Our classic integration for online payments. Current supported version: **v52**
+* [Recurring API](https://docs.adyen.com/api-explorer/#/Recurring/v49/overview): Endpoints for managing saved payment details. Current supported version: **v49**
+* [Payouts API](https://docs.adyen.com/api-explorer/#/Payout/v51/overview): Endpoints for sending funds to your customers. Current supported version: **v51**
+* [Platforms APIs](https://docs.adyen.com/platforms/api): Set of APIs when using Adyen for Platforms. 
+  * [Account API](https://docs.adyen.com/api-explorer/#/Account/v6/overview) Current supported version: **v6**
+  * [Fund API](https://docs.adyen.com/api-explorer/#/Fund/v6/overview) Current supported version: **v6**
+  * [Notification Configuration API](https://docs.adyen.com/api-explorer/#/NotificationConfigurationService/v1/overview) Current supported version: **v1**
+  * [Hosted Onboarding API](https://docs.adyen.com/api-explorer/#/Hop/v6/overview) Current supported version: **v6**
+* [Cloud-based Terminal API](https://docs.adyen.com/point-of-sale/terminal-api-reference): Our point-of-sale integration.
+* [BinLookup API](https://docs.adyen.com/api-explorer/#/BinLookup/v50/overview): Our integration for retrieving information, such as cost estimates, and 3D Secure supported version based on a given BIN.
+* [POS Terminal Management API](https://docs.adyen.com/api-explorer/#/postfmapi/v1/overview): Endpoints for managing your point-of-sale payment terminals: **v1** 
+* [Data Protection API](https://docs.adyen.com/development-resources/data-protection-api#): Endpoints to process Subject Erasure Requests as mandated in General Data Protection Regulation (GDPR): **v1** 
+* [Stored Value API](https://docs.adyen.com/payment-methods/gift-cards/stored-value-api): Endpoints to manage gift cards and other stored-value cards: **v46** 
+
+For more information, refer to our [documentation](https://docs.adyen.com/) or the [API Explorer](https://docs.adyen.com/api-explorer/).
+
+
+## Prerequisites
+ 
+* [Adyen test account](https://docs.adyen.com/get-started-with-adyen)
+* [API key](https://docs.adyen.com/development-resources/api-credentials#generate-api-key). For testing, your API credential needs to have the [API PCI Payments role](https://docs.adyen.com/development-resources/api-credentials#roles).
 * Java 8 or higher
 
 ## Installation
 
-You can use Maven or simply download the release.
-
-### Maven
-
-Add this dependency to your project's POM:
+You can use Maven and add this dependency to your project's POM:
```

Comment:
If there is something following you can use `:` otherwise only `.` or I am missing something :)